ecl/hql/hqllex.l

```diff
@@ -1459,6 +1461,23 @@ FUNCTIONMACRO|MACRO {
                                 complex = true;
                             else if (tok == '@')
                                 isFunctionMacro = true;
+                            else if (tok == UNKNOWN_ID && strieq(str(returnToken.queryId()), "EMBED"))
```

Comment:
more efficient to compare
returnToken.queryId()->queryLower() with createAtom("EMBED") { added to hqlatoms.* }